src/metadata.js

```diff
@@ -1,3 +1,9 @@
+function isObject(val) {
+  return val !== null
+    && val !== undefined
+    && (typeof val === 'function' || typeof val === 'object');
```

Comment:
whoops- the `&& val !== undefined` part isn't supposed to be there. It's a remnant from a perf test I was doing. `val && (type === 'function' || type === 'object')` or `val !== null && (type === 'function' || type === 'object')` will both work depending on whether you favor the strict or loose style.